src/server/assets/index.test.js

```diff
@@ -1,13 +1,25 @@
+/* eslint-disable global-require */
 import path from 'path';
-import getAssetsArray from '.';
+import nodeLogger from '../../app/helpers/logger.node';
+
+const mockLogError = jest.fn();
+
+jest.mock('../../app/helpers/logger.node', () => jest.fn());
+
+nodeLogger.mockImplementation(() => ({ error: mockLogError }));
 
 describe('getAssetsArray', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
   describe('no assets manifest', () => {
-    it('should console log an error', async () => {
+    it('should log an error', async () => {
       delete process.env.SIMORGH_ASSETS_MANIFEST_PATH;
-      global.console.log = jest.fn();
+      const getAssetsArray = require('./index.js').default;
       getAssetsArray();
-      expect(global.console.log).toHaveBeenCalledWith(
+      expect(mockLogError).toHaveBeenCalled();
```

Comment:
```suggestion
```

For `toHaveBeenCalledWith` to pass its implied that `toHaveBeenCalled` would pass